From 2488d31f5a946e63e40058baf29fd2991343ea6f Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Wed, 6 Dec 2017 13:14:22 -0500
Subject: adjust fopencookie structure tag for ABI-compat

stdio types use the struct tag names from glibc libio to match C++
ABI.
---
 include/stdio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/stdio.h b/include/stdio.h
index 2932c76..7c4f9ee 100644
--- a/include/stdio.h
+++ b/include/stdio.h
@@ -188,7 +188,7 @@ typedef ssize_t (cookie_write_function_t)(void *, const char *, size_t);
 typedef int (cookie_seek_function_t)(void *, off_t *, int);
 typedef int (cookie_close_function_t)(void *);
 
-typedef struct {
+typedef struct _IO_cookie_io_functions_t {
 	cookie_read_function_t *read;
 	cookie_write_function_t *write;
 	cookie_seek_function_t *seek;
-- 
cgit v0.11.2

